git_stalk/stalk.py

```diff
@@ -88,6 +90,7 @@ def get_details(event):
         "MemberEvent": "Added {0} as collaborator".format(member_login),
         "ReleaseEvent": "Released binaries for version {0}".format(release_tag_name),
         "ForkEvent": "Forked {0}".format(repo_name),
+        "DeleteEvent": "Deleted {0} {1}".format(deleted_obj_type, deleted_obj_ref),
```

Comment:
What if we change it to
`"DeleteEvent": "Deleted {1}: {0}".format(deleted_obj_type, deleted_obj_ref.title()),`